spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -98,6 +98,15 @@ def checkbox(self, name, options, defaultChecked = None):
 
     checkedIterables = self.z.checkbox(name, defaultCheckedIterables, optionIterables)
     return gateway.jvm.scala.collection.JavaConversions.asJavaCollection(checkedIterables)
+  
+  def registerCallback(self, event, cmd, replName="pyspark"):
```

Comment:
style? space in `replName = "pyspark"`
also, not sure if we should hardcode 'pyspark' here - this is configurable